docs/docs/tutorials/noirjs_app.md

```diff
@@ -1,122 +1,101 @@
+Feel free to use whatever versions, just keep in mind that Nargo and the NoirJS packages are meant to be in sync. For example, Nargo 0.19.x matches `noir_js@0.19.x`, etc.
 
-In this guide, we will be pinned to 0.17.0.
+In this guide, we will be pinned to 0.19.4.
 
 :::
 
-Make sure you have Node installed on your machine by opening a terminal and executing `node --version`. If you don't see a version, you should install [node](https://github.com/nvm-sh/nvm). You can also use `yarn` if you prefer that package manager over npm (which comes with node).
+Before we start, we want to make sure we have Node and Nargo installed.
 
-First of all, follow the the [Nargo guide](../getting_started/installation/index.md) to install nargo version 0.17.0 and create a new project with `nargo new circuit`. Once there, `cd` into the `circuit` folder. You should then be able to compile your circuit into `json` format and see it inside the `target` folder:
+We start by opening a terminal and executing `node --version`. If we don't get an output like `v20.10.0`, that means node is not installed. Let's do that by following the handy [nvm guide](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script).
 
-```bash
-nargo compile
+As for `Nargo`, we can follow the the [Nargo guide](../getting_started/installation/index.md) to install it. If you're lazy, just run this on a terminal:
+
+```sh
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
 ```
 
-Your folder structure should look like:
+## Our project
+
+ZK is a powerful technology. An app that doesn't reveal one of the inputs to *anyone* is almost unbelievable, yet Noir makes it as easy as a single line of code.
+
+In fact, it's so simple that it comes nicely packaged in `nargo`. Let's do that!
+
+### Nargo
+
+Run `nargo new circuit`.
+
+And... That's about it. Your program is ready to be compiled and run.
+
+To compile, let's `cd` into the `circuit` folder to enter our project, and call `nargo compile`. This compiles our circuit into `json` format and add it to a new `target` folder.
+
+:::info
+
+At this point in the tutorial, your folder structure should look like this:
 
 ```tree
 .
-└── circuit
+└── circuit <---- our working directory
     ├── Nargo.toml
     ├── src
     │   └── main.nr
     └── target
         └── circuit.json
 ```
 
-## Starting a new project
+:::
 
-Go back to the previous folder and start a new project by running run `npm init`. You can configure your project or just leave the defaults, and see a `package.json` appear in your root folder.
+### Node and Vite
 
-## Installing dependencies
+If you want to explore Nargo, feel free to go on a side-quest now and follow the steps in the [getting started](../getting_started/create_a_project.md) guide. However, we want our app to run on the browser, so we need Vite.
```

Comment:
This makes it sounds like you need Vite to run a Noir project in the browser, which technically isn't true. Consider updating to something like this:

```suggestion
If you want to explore Nargo, feel free to go on a side-quest now and follow the steps in the [getting started](../getting_started/create_a_project.md) guide. However, we want our app to run on the browser, so we can use Vite. Feel free to use [other frameworks](https://github.com/noir-lang/awesome-noir?tab=readme-ov-file#boilerplates).
```